samples/contrib/versioned-pipeline-ci-samples/mnist-ci-sample/README.md

```diff
@@ -0,0 +1,25 @@
+# Mnist CI Pipeline
+
+## What you can learn in this sample
+* CI process of a simple but general ML pipeline.
```

Comment:
Also mention that this CI is using cloud build service.

Reply:
OK.